.github/workflows/static.yml

```diff
@@ -22,6 +22,39 @@ jobs:
         run: |
           make fmt
           git diff --exit-code -- .
+  
+  duplimport:
+    name: Ensure there are no duplicate package imports
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Golang
+        uses: actions/setup-go@v3.2.0
+        with:
+          go-version-file: './backend/go.mod'
+      - name: "Install dupimport"
+        run: |
+          go install github.com/gostaticanalysis/dupimport/cmd/dupimport@latest
```

Comment:
Rather than `@latest`, can we use `@37d4ab484168ec60101dcc9bf644b54f910c2a25` (e.g. the most recent commit) to prevent new commits from breaking or otherwise subverting our build/test jobs?

Reply:
Done